dask/bag/core.py

```diff
@@ -151,6 +151,7 @@ def _to_textfiles_chunk(data, lazy_file, last_endline):
             endline = "\n"
             ensure = ensure_unicode
         else:
+            # TODO: This path is not being tested on bag/tests/test_bag.py yet
```

Comment:
I'm against adding these TODOs.  I think that coverage tools can give us this information separately from the codebase.

Reply:
I agree with this, although if someone wants to write up an issue to improve testing coverage in dask bag that'd be a nice new contributor issue.

Reply:
OK, I'll remove.